api/src/controllers/profile.py

```diff
@@ -0,0 +1,50 @@
+from flask import Blueprint, request
+from flask import jsonify
+
+from api.src.models.LXDModule import LXDModule
+from api.src.models.LXCProfile import LXCProfile
+from api.src.utils import response
+
+profile_api = Blueprint('profile_api', __name__)
+
+
+@profile_api.route('/')
+def profiles():
+    client = LXDModule()
+    return response.reply(client.listProfiles())
+
```

Comment:
Please handle the errors on this part also. Similar as you did with profile create.
It may crash if the client won't be able to connect to the LXD Daemon.